src/content/dataSubmissions/CrossValidation.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { FC, useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { useLazyQuery } from "@apollo/client";
+import { isEqual } from "lodash";
+import { Box, Button, Stack, styled } from "@mui/material";
+import { useSnackbar } from "notistack";
+import { Link } from "react-router-dom";
+import {
+  CrossValidationResultsInput,
+  CrossValidationResultsResp,
+  SUBMISSION_CROSS_VALIDATION_RESULTS,
+} from "../../graphql";
+import GenericTable, { Column } from "../../components/GenericTable";
+import { FormatDate, titleCase } from "../../utils";
+import ErrorDetailsDialog from "../../components/ErrorDetailsDialog";
+import QCResultsContext from "./Contexts/QCResultsContext";
+import { useSubmissionContext } from "../../components/Contexts/SubmissionContext";
+import CrossValidationFilters, {
+  FilterForm,
+} from "../../components/DataSubmissions/CrossValidationFilters";
+import StyledFormTooltip from "../../components/StyledFormComponents/StyledTooltip";
+
+const StyledErrorDetailsButton = styled(Button)({
+  display: "inline",
+  color: "#0B6CB1",
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontSize: "16px",
+  fontStyle: "normal",
+  fontWeight: 600,
+  lineHeight: "19px",
+  padding: 0,
+  textDecorationLine: "underline",
+  textTransform: "none",
+  "&:hover": {
+    background: "transparent",
+    textDecorationLine: "underline",
+  },
+});
+
+const StyledNodeType = styled(Box)({
+  display: "flex",
+  alignItems: "center",
+  textTransform: "capitalize",
+});
+
+const StyledSeverity = styled(Box)({
+  minHeight: 76.5,
+  display: "flex",
+  alignItems: "center",
+});
+
+const StyledBreakAll = styled(Box)({
+  wordBreak: "break-all",
+});
+
+const StyledIssuesTextWrapper = styled(Box)({
+  whiteSpace: "nowrap",
+  wordBreak: "break-word",
+});
+
+const columns: Column<CrossValidationResult>[] = [
+  {
+    label: "Batch ID",
+    renderValue: (data) => <StyledBreakAll>{data?.displayID}</StyledBreakAll>,
+    field: "displayID",
+    default: true,
+  },
+  {
+    label: "Node Type",
+    renderValue: (data) => <StyledNodeType>{data?.type}</StyledNodeType>,
+    field: "type",
+  },
+  {
+    label: "Submitted Identifier",
+    renderValue: (data) => <StyledBreakAll>{data?.submittedID}</StyledBreakAll>,
+    field: "submittedID",
+    sx: {
+      width: "20%",
+    },
+  },
+  {
+    label: "Conflicting Submission",
+    renderValue: ({ conflictingSubmission: _id }) => (
+      <span key={_id} data-testid={`conflicting-submission-${_id}`}>
+        <StyledFormTooltip title={_id}>
+          <Link
+            to={`/data-submission/${_id}`}
+            target="_blank"
+            data-testid={`conflicting-link-${_id}`}
+          >
+            ...
+            {_id?.slice(-5)}
```

Comment:
I see that an example resulting value is displayed as `...c54"]`. I wonder if the quotes and square bracket count towards the "last 5 digits" displayed as well, or if they meant just the id like: `...7fc54`. If your original way is correct then disregard this comment.

Reply:
Fixed in the submission `cfa56943-6e0f-4c9a-bac7-1b7c90313a2d`. This was a data issue due to how conflicting IDs used to be stored as a JSON string instead of an array.